<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>C_SyvDcExistingNodeWidget</class>
 <widget class="QWidget" name="C_SyvDcExistingNodeWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,1">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QGridLayout" name="gridLayout" columnstretch="0,1">
     <property name="leftMargin">
      <number>10</number>
     </property>
     <property name="topMargin">
      <number>9</number>
     </property>
     <property name="bottomMargin">
      <number>9</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="pc_LabelIcon">
       <property name="text">
        <string>Icon</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="stw_opensyde_gui_elements::C_OgeLabGroupSubItem" name="pc_LabelNodeId">
       <property name="text">
        <string>ID</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="stw_opensyde_gui_elements::C_OgeLabPropertiesSubHeading" name="pc_LabelName">
       <property name="text">
        <string>Name</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="stw_opensyde_gui_elements::C_OgeLabGroupSubItem" name="pc_LabelDeviceType">
       <property name="text">
        <string>Device</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="stw_opensyde_gui_elements::C_OgeGbxTransparent" name="pc_GroupBoxSerialNumber">
     <property name="minimumSize">
      <size>
       <width>190</width>
       <height>0</height>
      </size>
     </property>
     <property name="title">
      <string>GroupBox</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>1</number>
      </property>
      <property name="rightMargin">
       <number>1</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="stw_opensyde_gui::C_SyvDcExistingNodeDropAreaWidget" name="pc_WidgetSerialNumber" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>37</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>stw_opensyde_gui_elements::C_OgeGbxTransparent</class>
   <extends>QGroupBox</extends>
   <header>C_OgeGbxTransparent.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>stw_opensyde_gui_elements::C_OgeLabPropertiesSubHeading</class>
   <extends>QLabel</extends>
   <header>C_OgeLabPropertiesSubHeading.h</header>
  </customwidget>
  <customwidget>
   <class>stw_opensyde_gui::C_SyvDcExistingNodeDropAreaWidget</class>
   <extends>QWidget</extends>
   <header>C_SyvDcExistingNodeDropAreaWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>stw_opensyde_gui_elements::C_OgeLabGroupSubItem</class>
   <extends>QLabel</extends>
   <header>C_OgeLabGroupSubItem.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
